### PR TITLE
Adding files

### PR DESCRIPTION
**Pull request by amancdk**
---
Please check the following conditions for your PR.

- [ ] `apiDefinition.swagger.json` is validated using `paconn validate` command.
- [ ] `apiProperties.json` has a valid brand color. Invalid brand colors are `#007ee5` and `#ffffff`.

### CODE REVIEW

File: custom-connectors/EleadProductReference/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,3190 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Elead Product Reference Data",
+    "version": "1.0",
+    "description": "This API provides data to be used in conjunction with other eLeads APIs as needed.API searches, retrieves and provides the following details - Opportunity sources and sub sources for a specified company and up types, opportunity statuses for a specified company, sales steps for a specified company, employee positions for a specified company, employees by company and position, sender email addresses for a specified company, available vehicle class, year, make, model, trim.",
```
Comment (dougelliott) [inline custom-connectors/EleadProductReference/apiDefinition.swagger.json:6]: There is a missing space after `other eLeads APIs as needed` and the following sentence of `API searches, retrieves and provides the following`

**Review by dougelliott — CHANGES_REQUESTED** [review]
In addition to the comment, please have these files go into the `certified-connectors` directory, not the `custom-connectors` directory. Thanks!

File: custom-connectors/EleadProductReference/apiProperties.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#1F1A3C",
+    "capabilities": [],
+    "publisher": "Bhargava, Aman Avinash"
```
Comment (dougelliott) [inline custom-connectors/EleadProductReference/apiProperties.json:6]: Please also add the `stackOwner` property, and remove any PII. The publisher and stackowner fields should point to the company, not an individual, for certified connnectors.

**Review by dougelliott — CHANGES_REQUESTED** [review]

**Review by sriyen-msft — APPROVED** [review]

**Comment by amancdk** [comment]
@dougelliott & @sriyen-msft made required code review changes

**Review by dougelliott — APPROVED** [review]

**Pull request merged**